src/main/kotlin/com/atlassian/performance/tools/report/api/result/DurationData.kt

```diff
@@ -8,6 +8,14 @@ class DurationData(
     internal val durationMapping: (Double) -> Duration
 ) {
 
+    fun addStatsValue(value: Double) = stats.addValue(value)
+
+    fun statsValues(): DoubleArray? = stats.values
+
+    fun statsMean(): Double = stats.mean
+
+    fun statsN(): Long = stats.n
+
```

Comment:
Hmm, functionally it's just syntax sugar, but the original calls look more readable. What's the rationale behind those? Also `DescriptiveStatistics` has like 10 more members, why aren't they sugared?

Reply:
More then that if we are introducing new way to access `stats` values, maybe it's good to deprecate the `val stats` field of this class?

Reply:
Take a look at previous call chain which is because of having anemic model and kind of Train Wreck (anti) pattern:
`reader.read(baselineResult.actionMetrics)[label]?.stats?.values`

I searched for usages of stats field. Almost always nested field was used, not stats itself.
And the apache commons-math library is leaking more into the code (stats field)

In order to test it, you have to create and inject:
- reader
- stats
- values

After the change it's easier to test it and you don't use nested

Reply:
> Train Wreck (anti) pattern

What is that?
`reader.read(baselineResult.actionMetrics)[label]?.stats?.values`
vs
`reader.read(baselineResult.actionMetrics)[label]?.statsValues()`

Did this fix the pattern?

> I searched for usages of stats field. Almost always the last field was used, not stats itself.

Yep, but "almost" is the important part. This refactor missed the fact that `OutlierTrimming` consumes `stats.sortedValues`.

> And the apache commons-math library is leaking more into the code (stats field)

I agree. If that's the intention then as @pczuj said, we should deprecate the old API.
OTOH are we really that confident that we know all "legit" stats to consume? Are you sure that `stats.variance` is not of interest? Or `stats.geometricMean`? Is a copy of the `DescriptiveStatistics` interface actually cheaper to maintain?